person.rb

```diff
@@ -1,5 +1,8 @@
-class Person
+require_relative './nameable'
+
+class Person < Nameable
```

Comment:
- When I try to create a new person with 2 arguments, it gives me this error. Kindly fix it. Everything else if working fine.

![image](https://user-images.githubusercontent.com/92215878/223620721-fac09b47-182b-4148-96c2-f2734a72b095.png)